packages/design-system/src/components/Badge/Badge.test.tsx

```diff
@@ -1,42 +1,50 @@
 import React from 'react';
-import Badge from './Badge';
+import Badge, { BadgeProps } from './Badge';
```

Comment:
of all the components, i had the most trouble with badge. for some reason it's rendering multiple children within and i can't figure out why this component is so different compared to the others.

"Foo" appears as a direct child of `<ds-badge>` **and** as a child of the `<span class="ds-badge">` within. i don't know what it's behaving this way.